nmdc_server/ingest/common.py

```diff
@@ -47,3 +50,14 @@ def extract_extras(
         if key not in fields and key not in exclude:
             values["annotations"][key] = extract_value(value)
     return values
+
+
+def maybe_merge_download_artifact(ingest_db: Session, query):
+    logger = logging.getLogger()
+    for row in query:
+        try:
+            ingest_db.merge(row)
+            ingest_db.commit()
+        except IntegrityError:
+            logger.error("Error: data object with download history was removed.")
```

Comment:
```suggestion
            logger.info("Error: data object with download history was removed.")
```
For the sake of our alerting (to the extent that it exists), let's make this a `logger.info`. Upstream deletion shouldn't cause error alerts from our app.